packages/sveltekit/package.json

```diff
@@ -50,7 +50,7 @@
     "sorcery": "1.0.0"
   },
   "devDependencies": {
-    "@babel/types": "7.20.7",
+    "@babel/types": "^7.26.3",
```

Comment:
Pinning this appears to be unnecessary, and actually causes problems after the deduplication process.